metabench.cmake

```diff
@@ -377,6 +377,8 @@ file(WRITE ${CHART_HTML_ERB_PATH}
 "        /* configure the x axis */                                          \n"
 "        var xAxis = chart.xAxis || {};                                      \n"
 "        delete(chart.xAxis);                                                \n"
+"        if(typeof xAxis.axisLabel == 'undefined')                           \n"
+"          xAxis.axisLabel = 'Number of Elements'                            \n"
```

Comment:
> The X axis is not always a number of elements.

Well, this is just the default, we can still configure it to something else, which brings me back to #60. I think it's time we settle it before we add a bunch of unnecessary `chart.json`.

> this benchmark is broken as is

I wouldn't call it broken, but we really should rename it to `back`. BTW, now that the gh-pages is up and running, what about expanding our benchmark pool?